1.9/usage/pods/examples.md

```diff
@@ -0,0 +1,335 @@
+---
+post_title: Pods CLI
+menu_order: 30
+---
+
+# A pod with multiple containers
+	
+The following pod definition defines a pod with 3 containers.
```

Comment:
`The following pod definition specifies a pod...`. `Definition defines` reads a bit weird.